### ISSUE TITLE
Invalid using directives code fix with partially disabled warnings

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2026]
### Sample code

``` csharp
// Copyright (c) 2015 ACME, Inc. All rights reserved worldwide.

#if !VS2012

using System;
using System.ComponentModel;
using System.Runtime.InteropServices;
using System.Windows;
using System.Windows.Forms.Integration;
using System.Windows.Controls;
using System.Windows.Input;
using System.Windows.Interop;

#if VS2008
using Microsoft.Windows.Controls;
#elif VS2010
using System.Windows.Media;
#endif

using Forms = System.Windows.Forms;
using IWin32Window = System.Windows.Forms.IWin32Window;

namespace Microsoft.VisualStudio.Shell
{
#pragma warning disable SA1200 // Using directives must be placed correctly
    // This is required to work around accessibility issues in documentation comments.
    using NativeMethods = MyApplication.Namespace.NativeMethods;
#pragma warning restore SA1200 // Using directives must be placed correctly
}

#endif
```
### Configuration

``` json
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "documentationRules": {
      "fileNamingConvention": "metadata",
      "companyName": "ACME, Inc",
      "copyrightText": "Copyright (c) {year} {companyName}. All rights reserved worldwide.",
      "xmlHeader": false,
      "variables": {
        "year": "2015"
      }
    },
    "orderingRules": {
      "usingDirectivesPlacement": "outsideNamespace"
    }
  }
}
```
### Operation

Run an **Organize using directives** code fix for the SA1210 warning.
### Expected result

:memo: This bug report does not concern the blank lines which are removed from the result. Those are merely an accuracy issue, where I'm more concerned about the correctness issue.

``` csharp
// Copyright (c) 2015 ACME, Inc. All rights reserved worldwide.

#if !VS2012

using System;
using System.ComponentModel;
using System.Runtime.InteropServices;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Forms.Integration;
using System.Windows.Input;
using System.Windows.Interop;

#if VS2008
using Microsoft.Windows.Controls;
#elif VS2010
using System.Windows.Media;
#endif

using Forms = System.Windows.Forms;
using IWin32Window = System.Windows.Forms.IWin32Window;

namespace Microsoft.VisualStudio.Shell
{
#pragma warning disable SA1200 // Using directives must be placed correctly
    // This is required to work around accessibility issues in documentation comments.
    using NativeMethods = MyApplication.Namespace.NativeMethods;
#pragma warning restore SA1200 // Using directives must be placed correctly
}

#endif
```
### Actual result

``` csharp
// Copyright (c) 2015 ACME, Inc. All rights reserved worldwide.

#if !VS2012
#endif
#pragma warning disable SA1200 // Using directives must be placed correctly
using System;
using System.ComponentModel;
using System.Runtime.InteropServices;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Forms.Integration;
using System.Windows.Input;
using System.Windows.Interop;

#if VS2008
using Microsoft.Windows.Controls;
#elif VS2010
using System.Windows.Media;
using Forms = System.Windows.Forms;
using IWin32Window = System.Windows.Forms.IWin32Window;

namespace Microsoft.VisualStudio.Shell
{
// This is required to work around accessibility issues in documentation comments.
using NativeMethods = MyApplication.Namespace.NativeMethods;
#pragma warning restore SA1200 // Using directives must be placed correctly
}

#endif
```

**Comment by vweijsters** [comment]
Grabbing this
